src/Core/App.php

```diff
@@ -179,31 +179,45 @@ protected static function _classExistsInBase(string $name, string $namespace): b
      * Will return the path for tables `src/Model/Table`.
      *
      * @param string $type type of path
-     * @param string|null $plugin This argument is deprecated.
-     *   Use \Cake\Core\Plugin::classPath()/templatePath() instead for plugin paths.
-     * @return array
+     * @param string|null $plugin This argument is deprecated for class paths.
+     *   Use \Cake\Core\Plugin::classPath() instead or directly the method on \Cake\Core\Plugin.
+     * @return string[]
+     * @deprecated 4.0 Use \Cake\Core\App::classPath()/templatePath() instead.
```

Comment:
But there's not `App::templatePath()` currently.

Reply:
Do we want to create it? Or rather keep all lowercase "templates"/"locales" here in App::path()?